test/python/groundstation/list_plans_test.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2018 Infostellar, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The integration tests for ListPlans API in GroundStationService."""
+
+from datetime import datetime
+import unittest
+
+from google.protobuf.timestamp_pb2 import Timestamp
+from stellarstation.api.v1.groundstation import groundstation_pb2
+
+from conn.factory import StubFactory
+
+GS_ID = '27'
+
+
+class TestListPlans(unittest.TestCase):
```

Comment:
Can we use pytest? It has many useful features vs std library.

https://docs.pytest.org/en/latest/

Reply:
Thanks, changed to pytest and added very basic tests as the initial version.